sources/packages/web/src/views/aest/student/StudentRestrictions.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <content-group>
+    <v-row class="m-2">
+      <v-col class="category-header-medium color-blue">All Restrictions</v-col>
+      <v-col
+        ><v-btn class="float-right primary-btn-background"
+          ><font-awesome-icon icon="plus" class="mr-2" />Add restriction</v-btn
+        ></v-col
+      >
+    </v-row>
+    <DataTable
+      :value="studentRestrictions"
+      :paginator="true"
+      :rows="10"
```

Comment:
we have constants named DEFAULT_PAGE_LIMIT, PAGINATION_LIST for lines 14 and 15, you can use that,

Reply:
Feedback comment addressed.